src/Microsoft.TemplateEngine.Cli/CommandParsing/CommandParserSupport.cs

```diff
@@ -195,6 +195,7 @@ private static Option[] DebuggingCommandArgs
                     Create.Option("--debug:reset-config", string.Empty, Accept.NoArguments()),
                     Create.Option("--debug:showconfig", string.Empty, Accept.NoArguments()),
                     Create.Option("--debug:emit-timings", string.Empty, Accept.NoArguments()),
+                    Create.Option("--debug:enable-logging", string.Empty, Accept.NoArguments()),
```

Comment:
Let's check if there is a standard for this in dotnet CLI (or planned). One of the ways to do it can possibly be MSBuild style:

```
-v, --verbosity <LEVEL>   Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
```